iree/compiler/Dialect/VM/IR/VMOps.cpp

```diff
@@ -751,51 +697,9 @@ void ConstRefZeroOp::build(OpBuilder &builder, OperationState &result,
   result.addTypes(objectType);
 }
 
-static ParseResult parseRodataOp(OpAsmParser &parser, OperationState *result) {
-  // TODO(#4670): Share across ops or upstream a custom directive
-  StringRef visibility;
-  if (parser.parseOptionalKeyword(&visibility,
-                                  {"public", "private", "nested"})) {
-    parser.emitError(
-        parser.getCurrentLocation(),
-        "expected valid visibility specifier (public, private or nested)");
-    return failure();
-  }
-  StringAttr visibilityAttr = parser.getBuilder().getStringAttr(visibility);
-  result->attributes.push_back(parser.getBuilder().getNamedAttr(
-      SymbolTable::getVisibilityAttrName(), visibilityAttr));
-
-  StringAttr nameAttr;
-  Attribute valueAttr;
-  if (failed(parser.parseSymbolName(nameAttr,
-                                    mlir::SymbolTable::getSymbolAttrName(),
-                                    result->attributes)) ||
-      failed(parser.parseOptionalAttrDict(result->attributes)) ||
-      failed(parser.parseAttribute(valueAttr, "value", result->attributes))) {
-    return failure();
-  }
-
-  return success();
-}
-
-static void printRodataOp(OpAsmPrinter &p, RodataOp &op) {
-  p << op.getOperationName() << ' ';
-
-  // TODO(#4670): Share across ops or upstream a custom directive
-  StringRef visibilityAttrName = SymbolTable::getVisibilityAttrName();
-  if (auto visibility = op->getAttrOfType<StringAttr>(visibilityAttrName)) {
-    p << visibility.getValue() << ' ';
-  }
-
-  p.printSymbolName(op.sym_name());
-  p.printOptionalAttrDict(op->getAttrs(), /*elidedAttrs=*/{
-                              visibilityAttrName,
-                              "sym_name",
-                              "value",
-                          });
-  p << ' ';
-  p.printAttribute(op.value());
-}
+//===----------------------------------------------------------------------===//
+// custom<SymbolVisibility>($sym_visibility)
+//===----------------------------------------------------------------------===//
```

Comment:
should this block comment go up with `parseSymbolVisibility`?

Reply:
good ctach